package.json

```diff
@@ -28,7 +28,7 @@
     }
   },
   "peerDependencies": {
-    "webpack": ">=0.9 <2 || ^2.1.0-beta || ^2.2.0"
+    "webpack": ">=0.9 <2 || ^2.1.0-beta || >=2.2.0 <4"
```

Comment:
There was never a `2.0.x` the first release without `-beta` was `v2.2.0` to my knowledge. Yep for the `>= 0.9` if _really_ needed just  `>= 0.9 || ^1.0.0 || ^2.0.0 || ^3.0.0` then imho